Desktop-version changes

Marlyn-Mayienga opened the pull request:
# # Desktop-version changes
In this module, the following changes were made:
- made changes to the background images in the portfolio
- modified the desktop version to fit the design
- changed the styling to align the icons and text

Marlyn-Mayienga committed:
made changes to the desktop view:
added the background images to the desktop version
modified the styling to fit the desktop style

Marlyn-Mayienga committed:
fixed the html linter errors

Marlyn-Mayienga committed:
fix the html and css linter errors

keroloussamy commented on index.html:
```diff
@@ -25,95 +25,333 @@
-              <li>CodePen</li>
-              <li>GitLab</li>
-              <li>Terminal</li>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li>
+                <a href="#"><i class="far fa-envelope"></i></a>
+              </li>
```
- [ ] Kindly remove the underline from the nav links to match the Figma design.
- [ ] Kindly change the nav links color to white to match the Figma design.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166837396-c0777e4e-ac26-45eb-871c-b682a9a55419.png)


Your design
![image](https://user-images.githubusercontent.com/35535130/166837439-33c5b8f9-7c4d-4a9f-8311-173246530a39.png)

keroloussamy commented on index.html:
```diff
@@ -25,95 +25,333 @@
+          <hr class="first-line" />
+
+          <!-- PAGE BREAK -->
+          <section class="page-break">
+            <div>
+              <p>Work &darr;</p>
+            </div>
+            <div class="social-links">
+              <ul>
+                <li>
+                  <a href="https://github.com/Marlyn_Mayienga"
+                    ><i class="fab fa-github"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                    ><i class="fab fa-linkedin"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.instagram.com/merl.__/?hl=en"
+                    ><i class="fas fa-cat"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://twitter.com/M_ayienga"
+                    ><i class="fab fa-twitter"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="marlymayienga@gmail.com"
+                    ><i class="fas fa-envelope"></i
+                  ></a>
+                </li>
+              </ul>
+            </div>
+            <div>
+              <p>&darr; Work</p>
+            </div>
+          </section>
```
- [ ] Kindly change the color for the paragraph and the technologies to `#B3BAC5;` to match the Figma design.
- [ ] Kindy make the two buttons `Work` a little bit away from the accounts icons.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166837929-8263f158-dfdc-4046-86c1-00591433e282.png)


Your design
![image](https://user-images.githubusercontent.com/35535130/166837892-fbea03ab-7e86-4483-8157-a1f17c7c5748.png)

keroloussamy commented on style.css:
```diff
@@ -305,9 +329,15 @@ h3 {
 
-.contact-title.desktop {
+/* .content {
+  background: url("/circle_header.svg") no-repeat;
+  background-position: 150% 150%;
+  background-size: 200px;
+} */
+
+/* .contact-title.desktop {
   display: block;
-}
+} */
```
- [ ] Kindly remove the commented code, you can find it on the source tree always.

keroloussamy commented on index.html:
```diff
@@ -25,95 +25,333 @@
+                <img class="work-img" src="./project.jpg" alt="project image" />
+                <div class="work-content">
+                  <h4 class="work-item-title">Multi-Post Stories</h4>
+                  <p class="project-section">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
```
- [ ] Kindly change the color for the paragraph to `#B3BAC5` to match the Figma design.

keroloussamy commented on index.html:
```diff
@@ -25,95 +25,333 @@
+            <div id="names">
+              <input
+                type="text"
+                id="fname"
+                name="firstname"
+                placeholder="Enter First Name"
+                maxlength="30"
+                required
+              />
+              <input
+                type="text"
+                id="lname"
+                name="lastname"
+                placeholder="Last Name"
+                maxlength="30"
+                required
+              />
+            </div>
+            <input
+              type="email"
+              id="email"
+              name="Email"
+              placeholder="Email Address"
+              required
+            />
+
+            <textarea
+              id="msg"
+              name="user_message"
+              placeholder="Your message"
+              rows="10"
+              cols="50"
+              required
+              minlength="1"
+              maxlength="500"
+            ></textarea>
+
+            <button type="submit" class="send-button">Send</button>
+          </form>
+        </section>
```
- [ ] Kindly your mobile version still looks very good, just one thing you have to work on is adding some padding from left and right on the contact section to match the Figma design.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166838603-103994a0-a6fb-44fa-a255-08a39d80a194.png)

Your design
![image](https://user-images.githubusercontent.com/35535130/166838549-9112d06f-2db6-4d0e-9932-9fa03425f910.png)

keroloussamy reviewed (changes requested):
Hi @Marlyn-Mayienga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
made the following changes:
changed the colour of the p-tag of the text in the desktop version
changed the margin and padding of the icon section

gemmen29 reviewed (approved):
Hi @Marlyn-Mayienga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
